test/data/err_176.txt

```diff
@@ -0,0 +1,2 @@
+test/data/err_176.py:4:14 [FURB176]: Replace `utcnow` with `now(tz=timezone.utc)`
+test/data/err_176.py:5:12 [FURB176]: Replace `utcfromtimestamp` with `fromtimestamp(..., tz=timezone.utc)`
```

Comment:
I would turn `utcnow` into `utcnow()` to match the suggestion, same with `utcfromtimestamp`:

```
test/data/err_176.py:4:14 [FURB176]: Replace `utcnow()` with `now(tz=timezone.utc)`
test/data/err_176.py:5:12 [FURB176]: Replace `utcfromtimestamp(...)` with `fromtimestamp(..., tz=timezone.utc)`
```